sirepo/pkcli/roles.py

```diff
@@ -15,41 +15,45 @@
 import sirepo.server
 
 
-def add_roles(uid_or_email, *args):
+def add_roles(uid_or_email, *roles):
     """Assign roles to a user
     Args:
         uid_or_email (str): Uid or email of the user
         *args: The roles to assign to the user
     """
-    u = _check_uid_or_email_and_roles(uid_or_email, args)
-    sirepo.auth_db.UserRole.add_roles(u, args)
-    admin.audit_proprietary_lib_files(u)
+    sirepo.auth_db.UserRole.add_roles(
+        _args(uid_or_email, roles),
+        roles,
+    )
 
 
 
-def delete_roles(uid_or_email, *args):
+def delete_roles(uid_or_email, *roles):
     """Remove roles assigned to user
     Args:
         uid_or_email (str): Uid or email of the user
         *args: The roles to delete
     """
-    u = _check_uid_or_email_and_roles(uid_or_email, args)
-    sirepo.auth_db.UserRole.delete_roles(u, args)
-    admin.audit_proprietary_lib_files(u)
+    sirepo.auth_db.UserRole.delete_roles(
+        _args(uid_or_email, roles),
+        roles,
+    )
 
 
 def list_roles(uid_or_email):
     """List all roles assigned to a user
     Args:
         uid_or_email (str): Uid or email of the user
     """
-    u = _check_uid_or_email_and_roles(uid_or_email, [])
-    return sirepo.auth_db.UserRole.search_all_for_column('role', uid=u)
+    return sirepo.auth_db.UserRole.search_all_for_column(
+        'role',
+        uid=_args(uid_or_email, []),
+    )
 
 
 # TODO(e-carlin): This only works for email auth or using a uid
 # doesn't work for other auth methods (ex GitHub)
-def _check_uid_or_email_and_roles(uid_or_email, roles):
+def _args(uid_or_email, roles):
```

Comment:
check_args? I see that saying uid_or_email_and_roles is repetitive because those are the arg names. But _args seems too far to the terse side.

Reply:
I changed it to _parse_args. It's a private function that's used in all the other functions in the module. Therefore, it's critical to understand how it works, and the name is less use, since it does some critical functions: initializing, parsing, and validating. We could call it _preamble or _setup, but those are as unclear.